### DISCUSSION TITLE
I can't figure out how to setup encoding for my videos (1440p60) !

### DISCUSSION THREAD

**Discussion by OfficialMikeJ** [mediacms-io/mediacms#971] [Q&A]
I record my gameplays for YouTube in 1440p60. When I uploaded a video it didn't show in 1440p. I can't figure out or find the information I am looking for in regards to this. 

- Where is this information ?.
- How do you enable encoding ?.
- Which file paths and other settings do I need to setup ?.

I looked all through the github page but was not able too find or see anything in regards too this information.

any help would be awesome.

**Comment by u35253** [comment]
Maybe check here: https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#12-video-transcoding 

At the indicated location, `https://your_installation/admin/files/encodeprofile/` my default installation does have two 1440 profiles set up (h265 and vp9), but both are marked as Inactive.  Maybe flip one of them to Active.
